phlib/include/appresolver.h

```diff
@@ -14,23 +14,23 @@
 
 EXTERN_C_START
 
-HRESULT PhAppResolverGetAppIdForProcess(
+HRESULT NTAPI PhAppResolverGetAppIdForProcess(
```

Comment:
The AppResolver functions are not external or exported.

Reply:
3rd party projects which use the kphlib may link against un exported functions, and if they by default use a different calling convention then __stdcall a fix like this is required.